pkg/resources/schema_grant.go

```diff
@@ -92,43 +90,52 @@ func CreateSchemaGrant(data *schema.ResourceData, meta interface{}) error {
 	}
 
 	// ID format is <db_name>|<schema_name>||<privilege>
```

Comment:
can probably remove this comment now

Reply:
Should I remove all the comments like this?